workers/unity/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,14 +4,5 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes:
-  - enabled: 1
-    path: Assets/Playground/Scenes/ClientScene.unity
-    guid: 3f6e3d421576ca644b07380deacd35bb
-  - enabled: 1
-    path: Assets/Playground/Scenes/GameLogicScene.unity
-    guid: f5fda56aad5d91d408ff123b26ab6ade
-  - enabled: 1
-    path: Assets/Playground/Scenes/MobileClientScene.unity
-    guid: 82ccd0fd96a2ba24f88d0c7d01592f2e
+  m_Scenes: []
```

Comment:
Was it intentional to remove all the scenes? Just wanting to make sure!

Reply:
It was intentional - otherwise Unity would include all the scenes when trying to build the test player